install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -370,8 +370,14 @@ data:
 {{- if .Values.global.nodePort.range }}
   node-port-range: {{ .Values.global.nodePort.range | quote }}
 {{- end }}
+{{- if .Values.global.nodePort.devices }}
+  devices: {{ join " " .Values.global.nodePort.devices | quote }}
+{{- end }}
 {{- if .Values.global.nodePort.device }}
-  device: {{ join " " .Values.global.nodePort.device | quote }}
+  device: {{ .Values.global.nodePort.device | quote }}
```

Comment:
Should this just use `devices` since `--device` is deprecated?

Reply:
I don't have any strong opinion on this, but we currently merge both vals in `populateDevices()` in the agent for the backward compatibility. WDYT?

Reply:
> Shouldn't we just reject using both devices and device (in cilium-agent or in Helm)?

Yeah, makes sense. I will change.

Reply:
Resolved in: e361f6121a31140ba2fa9e77abcab6b852bf7014.